### PR TITLE
AWS Auto-Scaling closes #311

### LINKED ISSUE #311: AWS EC2 Auto Scaling Groups Module

### PR DESCRIPTION
**Pull request by Matt25969**
Just Task

### CODE REVIEW

File: topics/aws/modules/ec2-auto-scaling/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Overview
```
Comment (bob-crutchley) [inline topics/aws/modules/ec2-auto-scaling/README.md]:
```suggestion
# EC2 Auto-Scaling
```

File: topics/aws/modules/ec2-auto-scaling/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Auto-scaling is cool
+
+<!--TOC_START-->
+## Contents
+- [EC2 Auto-Scaling](#ec2-autoscaling)
+- [Tasks](#tasks)
+
+<!--TOC_END-->
+## EC2 Auto-Scaling
```
Comment (bob-crutchley) [inline topics/aws/modules/ec2-auto-scaling/README.md]:
```suggestion
## Overview
```

File: topics/aws/modules/ec2-auto-scaling/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+After another 2 - 5 minutes you should see that a new instance is now *Initializing*.  The Auto-Scaling group had noticed that the first instance was no long meeting its health checks and has created a new instance to fulfil the requirement.
+
+Navigate back to the **Auto Scaling Groups** dashboard
+
+Select the scaling group we created.
+
+At the bottom of this page, select the **Activity History** tab, read through the information in this tab and you will see how the Auto-Scaling Group has responded to the termination of the first instance. 
+
+Ensure that you delete the Auto-Scaling Group and all associated instances before you continue.
```
Comment (bob-crutchley) [inline topics/aws/modules/ec2-auto-scaling/README.md]: Could be split into more subheadings for readability.

**Comment by tvaidotas** [comment]
closing this for the time being

**Pull request closed without merging**